src/pyqasm/maps/constants.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 qBraid
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Constants for pyqasm maps."""
+
+DEFAULT_MAX_LOOP_ITERATIONS = 1000
```

Comment:
Let's change this to `1e10` (10 billion), as `1000` iterations are very limiting if not specified by the user